pkgs/misc/vscode-extensions/default.nix

```diff
@@ -570,6 +570,15 @@ let
 
       ms-vsliveshare.vsliveshare = callPackage ./ms-vsliveshare-vsliveshare {};
 
+      ms-vsliveshare.vsliveshare-pack = buildVscodeMarketplaceExtension {
+        mktplcRef = {
+          name = "vsliveshare-pack";
+          publisher = "ms-vsliveshare";
+          version = "0.4.0";
+          sha256 = "sha256-xTdfOqdypAaWpmtYIM0H7gwCy1jXNl5+HarvVW/3AiY=";
+        };
```

Comment:
Missing meta.

Reply:
[See here](https://github.com/NixOS/nixpkgs/pull/110461#issuecomment-765319578). I would add the license to the project but it isn't listed on spdx 🤔

Reply:
You can also add a license object here. Trex coping the definition from the licenses file, change the name and mark it unfree.